examples/relative_free_energy.py

```diff
@@ -103,7 +103,7 @@ def read_from_args():
     mol_b = get_mol_by_name(mols, args.mol_b_name)  # 30 in test pair
 
     print("Searching for the maximum common substructure...")
-    mcs_result = atom_mapping.mcs_map_graph_only_complete_rings(mol_a, mol_b)
+    mcs_result = atom_mapping.mcs(mol_a, mol_b, conformer_aware=False, timeout=60)
```

Comment:
I think this was an oversight, rather than intentional to use a different mcs mapping here. This is the old mapping prior to (https://github.com/proteneer/timemachine/pull/790/files#diff-83e3b80af4d75c2db550a5e4f928c4eb03e955ceacae8d6695e4e07cb1558df8).

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/812/commits/627816824c49d8f7d30631741a19b3a4386f33d5